packages/scandipwa/src/component/ProductReviews/ProductReviews.style.scss

```diff
@@ -71,8 +76,11 @@
         display: inline-block;
         margin-inline-start: 12px;
         margin-block: auto;
-        font-size: 24px;
-        font-weight: 700;
+        font-size: var(--h2-font-size);
+        font-weight: var(--h2-font-weight);
+        line-height: var(--h2-line-height);
+        text-transform: var(--h2-text-transform);
+        font-style: var(--h2-font-style);
```

Comment:
If base template changes h2 properties, should it also change Product Review Details ? Like I know there is project where h2 is 48px. Should those details also be so large ?

Reply:
Your questioning makes sense, we wouldn't want the size to become 48px.
What would be the best approach? Set strict properties according to our mock up? https://www.figma.com/file/qhlU4sg973vSMLexf3oiRM/SW%2FINT_PWA?node-id=4048%3A227

Reply:
Yes, if it is one case and since semantically it is not correct to set it as heading, you can just write own styles for this case.

Reply:
i just wanted to mention that these styles are not for the details they are for this title 
![Screenshot from 2021-12-06 09-18-43](https://user-images.githubusercontent.com/94956290/144803957-9b2af100-7f78-4f76-b1a7-1a0754382e89.png).

Reply:
Yes, we know